roles/user/tasks/subtasks/user_account.yml

```diff
@@ -85,6 +85,7 @@
     group: "{{ user.name }}"
     recurse: true
     follow: false
+  ignore_errors: true
```

Comment:
Why do you need this?

Reply:
i used vs sshed in the host proxmox to make the modification as root user, changes in the .git folder always results in an error. 

i guess it was a special use case.

a lot of changes i made here are kinda specific to my needs, however i'm not sure how to cherry pick the specific changes

Reply:
I'd recommend just skipping the user tag and handling that on your own rather than potentially break a warning that is useful to other users when same value is incorrect etc.